connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -104,6 +107,7 @@
     private InventoryCache inventoryCache;
     private ScoreboardCache scoreboardCache;
     private WindowCache windowCache;
+    private Map<Position, PlayerEntity> skullCache = new HashMap<>();
```

Comment:
Seems a bit expensive to store an entire player entity, especially if there are a bunch of skulls (such as in a staff hall). Is there any way you can reduce what is stored here and only store the data you need?